docs/historias_de_usuarios

```diff
@@ -0,0 +1,3 @@
+# Historias de usuarios
+
+[HU1] Voy último en un juego, que es la liga fantasy, que es un juego que estoy jugando con mis amigos, en el que formas una plantilla con once jugadores y estos jugadores obtienen puntos en función de lo que hagan en sus respectivos partidos en la vida real y gana el que al final de la liga consiga acumular más puntos. Me gustaría saber que jugadores debería fichar, teniendo en cuenta que dispongo de un presupuesto limitado de cien millones de euros,  para poder conseguir así más puntos e intentar mejorar mi situación.
```

Comment:
Tienes un problema similar a @javigp2002 [aquí](https://github.com/javigp2002/LazyFood/pull/2). No tienes suficientes detalles para poder desarrollar el producto. Pero es que cuantos más detalles hemos ido viendo (él lo ha desarrollado en un *user journey*), más complicado he visto el problema (o más simple, que es lo mismo).

Voy a intentar resumir.

1. Si el problema es que *cada jornada* o *al principio de la temporada* hay que fichar exactamente 11 jugadores (ni uno más ni uno menos) con 100 millones de euros, podría ser un problema interesante, pero dependerá de cómo estén organizadas las puntuaciones de los jugadores (que por lo que he entendido, son más o menos las mismas que el precio).
2. Si no es así, el reto es que lo que resuelve el problema sería contratar los jugadores que van a tener más puntuación para el precio que tienen *en el futuro*. Eso es un problema de predicción, **que no se puede abordar en la asignatura**.

Lo que le he sugerido al compañero es que piense en aspectos específicos (seleccionar un jugador para vender, optimizar la cláusula de rescisión de un jugador dependiendo de datos sobre el mismo, seleccionar el jugador más parecido a uno determinado a la hora de sustituir un jugador por otro en función de una serie de parámetros).

En todo caso, tanto @javigp2002 como tú tendréis que darle un par de vueltas para poder, a partir del contexto, que es el mismo, encontrar una serie de problemas específicos/historias de usuario, que si puede ser sean diferentes para cada uno.